src/main/resources/anet-schema.yml

```diff
@@ -710,6 +716,55 @@ properties:
           authorizationGroupRelatedObjects:
             "$ref": "#/$defs/labeledField"
 
+      eventSeries:
```

Comment:
Needs to be a required property

Reply:
All new dictionary fields are now required.